### PR TITLE
Add command-line args

### LINKED ISSUE #36: Main should have a framework for accepting command-line args and passing them as game configs

### PR DESCRIPTION
**Pull request by nathanshelly**
Only current accepted argument is `food-count` to specify the number of
pieces of food on-screen at once.

Closes #36

[asciicast](https://asciinema.org/a/JkRlWJ4EzElveKDeTFadtB6F5)

### CODE REVIEW

**Comment by nathanshelly** [comment]
Will merge this after #38 is merged since there will be some hairy conflicts there

File: Cargo.toml
Comment (sashaweiss) [inline Cargo.toml:4]: get outta here

File: src/snek/driver.rs
Comment (sashaweiss) [inline src/snek/driver.rs]: spacing

File: src/snek/driver.rs
Comment (sashaweiss) [inline src/snek/driver.rs:191]: spacing

File: src/snek/driver.rs
Comment (sashaweiss) [inline src/snek/driver.rs]: can we move the struct definitions to up above the `impl Driver`? Would prefer to have all the type definitions in one place, and all the impls in another

**Review by sashaweiss — APPROVED** [review]

**Pull request merged**
